save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -67,15 +70,18 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
         @RequestParam page: Int,
         @RequestParam size: Int,
         @RequestParam(required = false) status: TestResultStatus?,
-        @RequestParam(required = false) testSuite: String?,
+        @RequestParam(required = false) testFileName: String?,
+        @RequestParam(required = false) testSuiteName: String?,
+        @RequestParam(required = false) tag: String?,
```

Comment:
Why wouldn't you pass `TestExecutionFilters` annotated with `@RequestBody` (see how it is done in project, there are many examples)

Reply:
Done